examples/customWebWorkerTask/index.html

```diff
@@ -211,8 +210,9 @@ <h1>Example of using a custom web worker</h1>
         // dyanmically load the sleep task
         if(!sleepTaskLoaded) {
             sleepTaskLoaded = true;
+            // TODO: This URL probably won't work on hosted site
             cornerstoneWADOImageLoader.webWorkerManager.loadWebWorkerTask(
-                    '../examples/customWebWorkerTask/sleepTask.js',{
+              `${window.location.protocol}//${window.location.host}/examples/customWebWorkerTask/sleepTask.js`,{
```

Comment:
Someone learned a new trick.

Reply:
yeaaah but this is not great, I think the hosted example won't actually work but I didn't check.